pkg/acceptance/testdatatypes/testdatatypes.go

```diff
@@ -0,0 +1,10 @@
+package testdatatypes
+
+import "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/datatypes"
+
+var (
+	DataTypeNumber_36_2, _ = datatypes.ParseDataType("NUMBER(36, 2)")
```

Comment:
Nit: we can introduce something like `MustParseDataType`, similar to `regexp.MustCompile`

Reply:
We have 	SNOW-1843440 for it. I will add here the same TODO from other part of our codebase: `	// TODO [SNOW-1843440]: create using constructors (when we add them)?`